src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -3092,34 +3092,36 @@ public static void DCS_MyPersonSurrogate()
     [Fact]
     public static void DCS_FileStreamSurrogate()
     {
-        const string TestFileName = "Test.txt";
-        const string TestFileData = "Some data for data contract surrogate test";
+        using(var testFile = TempFile.Create())
+        {
+            const string TestFileData = "Some data for data contract surrogate test";
 
-        // Create the serializer and specify the surrogate
-        var dcs = new DataContractSerializer(typeof(MyFileStream));
-        dcs.SetSerializationSurrogateProvider(MyFileStreamSurrogateProvider.Singleton);
+            // Create the serializer and specify the surrogate
+            var dcs = new DataContractSerializer(typeof(MyFileStream));
+            dcs.SetSerializationSurrogateProvider(MyFileStreamSurrogateProvider.Singleton);
 
-        // Create and initialize the stream
-        byte[] serializedStream;
+            // Create and initialize the stream
+            byte[] serializedStream;
 
-        // Serialize the stream
-        using (MyFileStream stream1 = new MyFileStream(TestFileName))
-        {
-            stream1.WriteLine(TestFileData);
-            using (MemoryStream memoryStream = new MemoryStream())
+            // Serialize the stream
+            using (MyFileStream stream1 = new MyFileStream(testFile.Path))
```

Comment:
nit, if you want, we usually use 'var' when the assigned type is obvious (ie a constructor)